### ISSUE TITLE
Why use https URL to be wrong?

### ISSUE THREAD

**Issue by hardway120308** [zhengchun/feedocean#1]
Why use https URL to be wrong?Error Message:

Swapping to Development environment will display more detailed information about the error that occurred.

Development environment should not be enabled in deployed applications, as it can result in sensitive information from exceptions being displayed to end users. For local debugging, development environment can be enabled by setting the ASPNETCORE_ENVIRONMENT environment variable to Development, and restarting the application.

**Comment by zhengchun** [comment]
Hello, Maybe this post can help you [Use multiple environments in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/environments?view=aspnetcore-2.1)

**Comment by hardway120308** [comment]
No, I am saying that an error occurs when entering a URL with "https://" in FeedOcean

**Comment by zhengchun** [comment]
Sorry , I test is OK(Chrome 67 and Edge), what is you web browser and version?

**Comment by hardway120308** [comment]
My browser is also Chrome 67, the URL of my generated feed is [https://bnext.com.tw]("https://bnext.com.tw")

Screenshots：

![feedocean-issues-1](https://user-images.githubusercontent.com/38828904/43781033-93c813dc-9a8e-11e8-88eb-648aa0402919.png)

![feedocean-issues-2](https://user-images.githubusercontent.com/38828904/43781088-b208e736-9a8e-11e8-82da-2fd81f823a81.png)

**Comment by zhengchun** [comment]
Thanks report, This issue has been fixed. You can try again!

**Comment by hardway120308** [comment]
Thank you! I can use it now!

Thank you again!

**Comment by rhefirst1** [comment]
Hello
This probldm happend second time from 2 days 
Any help to fix it please
